src/Render/md.jl

```diff
@@ -0,0 +1,120 @@
+["Markdown rendering."]
+
+Formats.parsedocs(::Formats.Format{Formats.MarkdownFormatter}, raw, m, obj) = Markdown.parse(raw)
+
+## Prepare methods
+type RenderedMarkdown
+    outdir      :: UTF8String
+    outpages    :: Vector{Tuple{AbstractString, AbstractString}}
+    layout      :: Vector{Tuple{UTF8String, Vector}}
```

Comment:
Would these two ``Vector{Tuple{...}}`` be better as ``Dict{String, String}``? Do they need to be ordered?